test/runtests.jl

```diff
@@ -58,8 +58,8 @@ using Base.Test, Missings
     @test zero(Union{Int, Missing}) === 0
     @test zero(Union{Float64, Missing}) === 0.0
 
-    for T in (subtypes(Dates.DatePeriod)...,
-              subtypes(Dates.TimePeriod)...)
+    for T in (subtypes(Compat.Dates.DatePeriod)...,
```

Comment:
Looks like you need `using Compat` at the top of the file.

Reply:
Thanks.